resolwe/flow/execution_engines/bash/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""An execution engine that outputs bash programs."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from django.conf import settings
+
+from resolwe.flow.execution_engines import BaseExecutionEngine, ExecutionError
```

Comment:
I don't know. To be safe I prefer base classes in base.py, exceptions in exceptions.py and in __init__.py just import what I need in the namespace.